schema/vrs.yaml

```diff
@@ -176,35 +186,13 @@ definitions:
       subject:
         oneOf:
           - $ref: "#/definitions/MolecularVariation"
-          - $ref: "#/definitions/CURIE"
```

Comment:
@ahwagner We need to leave CURIE here, otherwise we can't use computed ids as subjects.

Reply:
Good catch! This should be reverted.